#VRML_SIM R2019a utf8
WorldInfo {
}
Viewpoint {
  orientation 0.5267517123557107 -0.8168219746001958 -0.23523285344646006 5.282543013830838
  position 1.3447372072140014 0.8025827575209821 0.890335242492693
}
TexturedBackground {
}
TexturedBackgroundLight {
}
Floor {
}
Robot {
  translation 0 0.08499999999999998 0
  children [
    DEF GR_Robot Group {
      children [
        Shape {
          appearance PBRAppearance {
            baseColor 0.9372549019607843 0.1607843137254902 0.1607843137254902
          }
          geometry DEF G_BOX Box {
            size 0.25 0.07 0.1
          }
        }
        Transform {
          translation 0.11 -0.05 0
          children [
            Shape {
              appearance PBRAppearance {
                baseColor 0.4470588235294118 0.6235294117647059 0.8117647058823529
                roughness 1
                metalness 0
              }
              geometry DEF CRY_wheel Sphere {
                radius 0.015
              }
            }
          ]
        }
      ]
    }
    HingeJoint {
      jointParameters HingeJointParameters {
        axis 0 0 1
        anchor -0.05 0 0.06
      }
      device [
        PositionSensor {
          name "position_sensor1"
          resolution 1
        }
        RotationalMotor {
          name "motor_right"
          maxVelocity 52.35
          maxTorque 0.593
        }
      ]
      endPoint Solid {
        translation -0.049336666060276646 0.0013762586138605132 0.059995405976255904
        rotation -0.386125271396529 -0.6520030938036283 -0.652532942049253 3.8791036193607566
        children [
          Shape {
            appearance DEF A_W PBRAppearance {
              baseColor 0 0 0
            }
            geometry DEF G_W Cylinder {
              height 0.02
              radius 0.075
            }
          }
        ]
        name "solid(1)"
        boundingObject USE G_W
        physics Physics {
        }
      }
    }
    HingeJoint {
      jointParameters HingeJointParameters {
        axis 0 0 1
        anchor -0.05 0 -0.06
      }
      device [
        RotationalMotor {
          name "motor_left"
          maxVelocity 52.35
          maxTorque 0.593
        }
      ]
      endPoint Solid {
        translation -0.04929989700044014 0.0014102528753755154 -0.05999909719608424
        rotation 0.9861664507086423 0.11715868973095092 0.11725857289446187 1.5839411369496914
        children [
          Shape {
            appearance USE A_W
            geometry USE G_W
          }
        ]
        boundingObject USE G_W
        physics Physics {
        }
      }
    }
    DistanceSensor {
      translation 0.125 0 0
      rotation 0 0 1 0.07
      children [
        Shape {
          appearance PBRAppearance {
            baseColor 0.45098039215686275 0.8235294117647058 0.08627450980392157
          }
          geometry Box {
            size 0.02 0.01 0.01
          }
        }
      ]
      name "distance_sensor"
      lookupTable [
        0 0 0
        0.02 0 0
        5 1023 0
      ]
      resolution 1
    }
  ]
  boundingObject USE GR_Robot
  physics Physics {
    density 1300
  }
  controller "u2_exam"
}
Solid {
  translation 0.6200000000000003 0.1 0
  children [
    Shape {
      appearance PBRAppearance {
        baseColor 0.4470588235294118 0.6235294117647059 0.8117647058823529
      }
      geometry DEF G_BOX Box {
        size 0.2 0.2 0.2
      }
    }
  ]
  boundingObject USE G_BOX
  physics Physics {
  }
}
